API Blueprint Specification.md

```diff
@@ -478,6 +488,189 @@ Name and description of the API
 
 ---
 
+<a name="def-authentication-section"></a>
+## Authentication section
+- **Parent sections:** none, [Action section](#def-action-section), [Request section](#def-request-section)
+- **Nested sections:** [`1` Authentication schemes section](#def-authenticationschemes-section), [`1` Response section](#def-response-section)
+- **Markdown entity:** header, list
+- **Inherits from**: none
+
+#### Definition
+Defined by the `Authentication` keyword in Markdown header entity.
+
+    # Authentication
+
+**-- or --**
+
+Defined by the `Authentication` keyword in Markdown list entity followed by optional comma seperated list of scheme names which is prefixed by a colon.
+
+    + Authentication: <list>
+
+> **NOTE:** The second notation is only used when the parent section is either an [Action section](#def-action-section) or a [Request section](#def-request-section).
+
+#### Description
+Based on the parent section, the usage of this section differs as shown below.
```

Comment:
Overall description instead of just 

> Based on the parent section, the usage of this section differs as shown below.

Something along these lines: 

> This section does/represents XXXX